include/clad/Differentiator/BuiltinDerivatives.h

```diff
@@ -144,6 +144,8 @@ CUDA_HOST_DEVICE inline ValueAndPushforward<double, double>
 __builtin_pow_pushforward(double x, double exponent, double d_x,
                           double d_exponent) {
   auto val = __builtin_pow(x, exponent);
+  if (exponent == 0 && d_x == 1)
```

Comment:
```suggestion
  if (exponent == 0 && d_exponent == 0)
```
You assume that it will be either `d_x == 1 && d_exponent == 0`, or `d_x == 0 && d_exponent == 1`. But I'm afraid this might not generalize, if `d_x` might also be a non-zero value that is not `1`. @vgvassilev, can this happen?

Maybe it's better to do the check `d_exponent == 0` instead? Also in the other pushforwards.

Reply:
> @vgvassilev, can this happen?

No idea, and I'd suggest if we are not sure to be conservative and assume it might happen..